openwisp_users/models.py

```diff
@@ -25,6 +26,24 @@ class Organization(BaseOrganization, AbstractOrganization):
     class Meta(AbstractOrganization.Meta):
         swapper.swappable_setting('openwisp_users', 'Organization')
 
+    def add_user(self, user, **kwargs):
+        """
+        overriding this method of ``AbstractOrganization``
+        to remove organization owner creation logic as it is
+        being handled separately by a signal of this project.
```

Comment:
Let's shorten this docstring, something like the following will work:

We override this method from the upstream dependency to
skip the creation of the organization owner, which we perform
automatically via a signal receiver.
Without this change, the add_user method would throw IntegrityError.